docs/conf.py

```diff
@@ -131,13 +132,24 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_logo = "./_static/with-text-light-190px.png"
+html_theme_options = {
+    "logo_only": True,
+    #
+    # TOC options
+    #   https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#theme-options
+    "includehidden": False,
```

Comment:
I was going to ask what the reasoning behind this is, but I see later on that it allowed you to refactor the formulary index table, for which, :+1: !

Reply:
That, and it allows use to use the `nbgallery` directive in `./docs/index.rst` without the gallery titles showing up in the sidebar.

Reply:
OH SO THAT'S IT!!!!!!!